artman/tasks/python_grpc_tasks.py

```diff
@@ -143,29 +151,37 @@ def _copy_proto(self, src, dest, common_protos):
                     if import_:
                         dest_file.write('import "{}";\n'.format(
                             self._transform(
-                                import_.group('package'), '/', common_protos)))
+                                import_.group('package'), '/', common_protos,
+                                api_name=api_name, api_version=api_version,
+                            ),
+                        ))
                     else:
                         dest_file.write(line)
 
     def _copy_and_transform_directories(
             self, src_directories, destination_directory, common_protos,
-            paths=None):
+            api_name, api_version, paths=None):
         for path in src_directories:
             protos = list(protoc_utils.find_protos([path], []))
             for proto in protos:
                 src_base_dirs = self._extract_base_dirs(proto)
                 sub_new_src = os.path.join(
                     destination_directory,
                     self._transform(
-                        src_base_dirs, os.path.sep, common_protos))
+                        src_base_dirs, os.path.sep, common_protos,
+                        api_name=api_name, api_version=api_version,
+                    ),
+                )
                 if paths is not None:
                     paths.add(sub_new_src)
 
                 dest = os.path.join(sub_new_src, os.path.basename(proto))
                 if not os.path.exists(dest):
                     self.exec_command(['mkdir', '-p', sub_new_src])
                 self._copy_proto(
-                    proto, os.path.join(sub_new_src, dest), common_protos)
+                    proto, os.path.join(sub_new_src, dest), common_protos,
+                    api_name=api_name, api_version=api_version,
```

Comment:
nit: inconsistent to use kwargs only for this change?

Reply:
Correct; I was in a hurry and not going back and figuring out what everything else on these private methods were.